src/modules/lab/pages/library/LibraryOverview.tsx

```diff
@@ -29,22 +28,9 @@ export default function LibraryOverviewPage() {
         <div className='font-bold xl:pt-3 pb-3 text-lg'>Library Details</div>
         <LibraryTableDetails libraryDetail={libraryDetailRes} />
 
-        <Link
-          to={`/runs/workflow?search=${
```

Comment:
Initially, it redirected to the workflow table filtered by a library, but it is just pasting this as is in the search box. So not sure about the accuracy by doing this, perhaps could re-enable the link if the workflow table is able to filtered by library.

Reply:
```
# Combine search across multiple fields (worfkflow run name, comment, library_id, orcabus_id, workflow name)
if search_params:
            result_set = result_set.filter(
                Q(workflow_run_name__icontains=search_params) |
                Q(comment__icontains=search_params) |
                Q(libraries__library_id__icontains=search_params) |
                Q(libraries__orcabus_id__icontains=search_params) |
                Q(workflow__workflow_name__icontains=search_params)
).distinct() # Add distinct to remove duplicates
```

in the backend, we filter by library id like L2401535. The metadata manager library id seems not match in the workflow manger db

Reply:
Right, but I guess in this case the `orcabus_id` prefix should be stripped in order to work.

Reply:
yes, you are right, I will check again with new orcabus id backend

Reply:
yes, you are right, I will check it again with new orcabus id backend